godtools/App/Features/Onboarding/Presentation/OnboardingTutorial/OnboardingTutorialView.swift

```diff
@@ -75,11 +75,11 @@ struct OnboardingTutorialView: View {
                 FixedVerticalSpacer(height: 30)
                 
                 PageControl(numberOfPages: 4, attributes: GTPageControlAttributes(), currentPage: $viewModel.currentPage)
+                    .flipsForRightToLeftLayoutDirection(true)
```

Comment:
Same here.  Might be redundant adding the modifier here since it's already included on the PageControl top VStack view. -> https://github.com/CruGlobal/godtools-swift/blob/develop/godtools/App/Share/SwiftUI%20Views/PageControl/PageControl.swift#L63